scan/src/reusable/InfoHL.re

```diff
@@ -128,14 +128,21 @@ let make = (~info, ~header, ~isLeft=true) => {
           }}
        </div>
      | DataSources(ids) =>
-       <div className=Styles.datasourcesContainer>
-         {ids
-          ->Belt.List.map(id =>
-              <> <TypeID.DataSource id position=TypeID.Subtitle /> <HSpacing size=Spacing.sm /> </>
-            )
-          ->Array.of_list
-          ->React.array}
-       </div>
+       switch (ids |> Belt_List.size) {
+       | 0 => <Text value="Undetermined" size=Text.Lg spacing={Text.Em(0.06)} height=Text.Px(17)/>
```

Comment:
TBD

Reply:
Ok, I'll edit it on the next PR. Thanks to informing me.